pkg/infrastructure/data_index.go

```diff
@@ -26,11 +26,11 @@ import (
 
 const (
 	// DefaultDataIndexImageFullTag is the default image name for the Kogito Data Index Service
-	DefaultDataIndexImageFullTag = "quay.io/kiegroup/kogito-data-index:latest"
+	DefaultDataIndexImageFullTag = "quay.io/kiegroup/" + DefaultDataIndexImageName + ":latest"
 	// DefaultDataIndexImageName is just the image name for the Data Index Service
 	DefaultDataIndexImageName = "kogito-data-index"
```

Comment:
I just noticed it now, but, wouldn't make sense put the image name const before the full tag?